### PR TITLE
Add jobs listing and cancel

### PR DESCRIPTION
**Pull request by jchyzy**

### CODE REVIEW

**Comment by pswedrak** [comment]
The functionality is ok and the code is fine but gui looks horrible. 
image

**Comment by jchyzy** [comment]
On my computer it looked ok, but I improve it a little bit so I hope it resolves the problem.

File: gui/functionality/manage_jobs.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from idact import show_cluster, load_environment
+from idact.detail.slurm.run_scancel import run_scancel
+from idact.detail.slurm.run_squeue import run_squeue
+
+from gui.functionality.idact_app import IdactApp, WindowType
+from gui.helpers.decorators import addToClass
+from gui.helpers.worker import Worker
+
+
+class ManageJobs:
```
Comment (PiekarzKarolina) [inline gui/functionality/manage_jobs.py]: I'm not sure about naming of this class. I think it should be more like `JobManager` or something similar.

Comment (jchyzy) [reply gui/functionality/manage_jobs.py]: Other classes are named similarly - `AddCluster`, `RemoveCluster`. So as it should be consistent we should rename all classes or left it as it is.

Comment (PiekarzKarolina) [reply gui/functionality/manage_jobs.py]: Oh, good point. So I think it should stay as it is.

Comment (PiekarzKarolina) [reply gui/functionality/manage_jobs.py]: Oh, good point. So I think it should stay as it is.

**Comment by PiekarzKarolina** [comment]
After typing random job number it shows a success message even if that job didn't exist at all.

**Comment by jchyzy** [comment]
> After typing random job number it shows a success message even if that job didn't exist at all.
Yes, I know that. But as I saw the command that is executed in idact does not give any message if the job exist or not - it just runs the command.

**Pull request merged**